src/groth16/aggregate/verify.rs

```diff
@@ -85,71 +85,74 @@ pub fn verify_aggregate_proof<E: Engine + std::fmt::Debug, R: rand::RngCore + Se
         // NOTE: From this point on, we are only checking *one* pairing check so
         // we don't need to randomize as all other checks are being randomized
         // already so this is the "base check" so to speak.
-        pairing_checks_copy.merge_parts(
+        pairing_checks_copy.merge_pairing_equation(
             // 3. Compute left part of the final pairing equation
-            move || {
```

Comment:
could just use `par!` before to generate these in parallel, and then merge them in